modules/sdk-evm/src/types.ts

```diff
@@ -0,0 +1,17 @@
+export const supportedCoins: string[] = ['eth', 'bsc', 'polygon', 'gteth', 'tbsc', 'tpolygon'];
```

Comment:
this looks like it might be better as an array of `CoinFamily`.

```suggestion
import { CoinFamily } from '@bitgo/statics';
export const supportedCoins: CoinFamily[] = [CoinFamily.ETH, CoinFamily.BSC, CoinFamily.POLYGON]
```